.github/workflows/deploy-dev.yaml

```diff
@@ -1,117 +1,135 @@
-name: Deploy Dev
+name: Deploy DUP-Public Dev
+
+run-name: Deploying ${{ github.ref_name }} (Public) to dev
 
 on:
   push:
     branches: [main]
-
-env:
-  TF_VERSION: 0.14.7
-  TG_VERSION: 0.37.1
-  TG_SRC_PATH: terraform
-  TFC_WORKSPACE: dev
-  TARGET_ENV: dev
+    paths: "dup-public/**"
```

Comment:
Is this a real path?

Reply:
Replied on the other PR but just incase its ever relevant to someone in the future. 

 Originally all the SAM files were in a dup-admin/ dir and when I pulled out terraform moved all the SAM stuff up one level.

Ill make the updates.

Reply:
Resolved in: [commit](https://github.com/bcgov/parks-reso-public/pull/452/commits/c4ec13f97b6fb7a4b34a10179d75c0320646dcec)